data/json/obsolete.json

```diff
@@ -1639,5 +1639,20 @@
       [ "VISE", 1 ]
     ],
     "use_action": [ "HAMMER", "CROWBAR" ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "knuckle_scrap",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "skills_required": [ "bashing", 1 ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" } ],
+    "difficulty": 2,
+    "time": "30 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [ [ [ "rag", 4 ] ], [ [ "scrap", 4 ] ] ]
```

Comment:
Actually, I just realized that this needs to be moved to `data/json/recipe/recipe_obsolote.json`, in the following form:
```json
{
  "type": "recipe",
  "result": "knuckle_steel",
  "obsolete": true
}
```
Sorry for the hassle.

Reply:
nah, it's fine. It helps me learn what to do in the future so it becomes less of a hassle overall. I'm assuming I should delete that file update

Reply:
Yeah, the changes to `data/json/obsolete.json` is not needed.